test/metabase/query_processor_test/explicit_joins_test.clj

```diff
@@ -261,7 +261,7 @@
                rows))))))
 
 (deftest select-*-source-query-test
-  (mt/test-drivers (mt/normal-drivers-with-feature :left-join)
+  (mt/test-drivers (mt/normal-drivers-with-feature :left-join :foreign-keys)
```

Comment:
Same as what I was saying above: BigQuery doesn't technically have foreign keys, but there is nothing stopping you from joining a table using regular columns. So we shouldn't disable this test for BigQuery.

Is there a reason MongoDB can't do this join? Because it's actually using `:right-join`? I think `mt/normal-drivers-with-feature` is just wrong here. This is what it should be:

```suggestion
  (mt/test-drivers (mt/normal-drivers-with-feature :right-join)
```

Reply:
The same as with the pivot tests: preprocessing requires foreign key support.